server/api/controllers/auth/post-signup.js

```diff
@@ -35,6 +35,13 @@ the account verification message.)`,
       description: "The user's full name.",
     },
 
+    nationality: {
+      required: true,
+      type: "string",
+      example: "Hong Kong",
```

Comment:
wondering what the datas would look like in the future for this field. "hong_kong"? "hongkong"? will this be a drop down/text field? if drop down do we have a table of countries? not a big deal right now but just came across my mind as i read through this

Reply:
yeah good q. i thought about that too and thinking more like a drop down table for frontend. is it better if its a table of countries or an array of strings stored as constant?